python/sglang/srt/custom_op.py

```diff
@@ -40,6 +41,8 @@ def dispatch_forward(self):
             return self.forward_cuda
         elif _is_hip:
             return self.forward_hip
+        elif not torch.cuda.is_available() and cpu_has_amx_support():
```

Comment:
If cuda is available but the device is set to cpu, do we want to support this use case?